views/js/back.js

```diff
@@ -0,0 +1,446 @@
+
+    $(document).on('click', '.modify_icon', function () {
+        var id = $(this).data('id');
+        $('#show_icon_'+id).removeClass('inactive');
+        $('#show_icon_'+id).addClass('active');
+    });
+
+    $(document).on('click', '.select-icon', (e) => {
+        var id = $(e.target).data('id');
+        $('#show_icon_'+id).removeClass('active');
+        $('#show_icon_'+id).addClass('inactive');
+        $('.show-rea-block.active .psr-picto').attr('data-image', $(e.target)[0]['innerHTML']);
+        $('.show-rea-block.active .psr-picto i').html($(e.target)[0]['innerHTML']);
+    });
+
+    $(document).on('change', 'input[name="PSR_REDIRECTION_1"]', (e) => {
+        if($(e.target).val() == 0) {
+            $('.psr-cms').removeClass('active');
+            $('.psr-url').removeClass('active');
+            $('.psr-cms').addClass('inactive');
+            $('.psr-url').addClass('inactive');
+        } else if ($(e.target).val() == 1) {
+            $('.psr-url').removeClass('active');
+            $('.psr-url').addClass('inactive');
+            $('.psr-cms').removeClass('inactive');
+            $('.psr-cms').addClass('active');
+        } else if ($(e.target).val() == 2) {
+            $('.psr-cms').removeClass('active');
+            $('.psr-cms').addClass('inactive');
+            $('.psr-url').removeClass('inactive');
+            $('.psr-url').addClass('active');
+        }
+    });
+
+    $(document).on('change', 'input[name="PSR_REDIRECTION_2"]', (e) => {
+        if($(e.target).val() == 0) {
+            $('.psr-cms').removeClass('active');
+            $('.psr-url').removeClass('active');
+            $('.psr-cms').addClass('inactive');
+            $('.psr-url').addClass('inactive');
+        } else if ($(e.target).val() == 1) {
+            $('.psr-url').removeClass('active');
+            $('.psr-url').addClass('inactive');
+            $('.psr-cms').removeClass('inactive');
+            $('.psr-cms').addClass('active');
+        } else if ($(e.target).val() == 2) {
+            $('.psr-cms').removeClass('active');
+            $('.psr-cms').addClass('inactive');
+            $('.psr-url').removeClass('inactive');
+            $('.psr-url').addClass('active');
+        }
+    });
+
+    $(document).on('change', 'input[name="PSR_REDIRECTION_3"]', (e) => {
+        if($(e.target).val() == 0) {
+            $('.psr-cms').removeClass('active');
+            $('.psr-url').removeClass('active');
+            $('.psr-cms').addClass('inactive');
+            $('.psr-url').addClass('inactive');
+        } else if ($(e.target).val() == 1) {
+            $('.psr-url').removeClass('active');
+            $('.psr-url').addClass('inactive');
+            $('.psr-cms').removeClass('inactive');
+            $('.psr-cms').addClass('active');
+        } else if ($(e.target).val() == 2) {
+            $('.psr-cms').removeClass('active');
+            $('.psr-cms').addClass('inactive');
+            $('.psr-url').removeClass('inactive');
```

Comment:
You use a lot the combination `X.removeClass('inactive'); X.addClass('active');`
You dont want to make it a private reusable function 😄?

Reply:
Is it very necessary ?